apps/desktop/src/lib/baseBranch/baseBranch.test.ts

```diff
@@ -0,0 +1,178 @@
+import { BaseBranch } from '$lib/baseBranch/baseBranch';
+import { describe, expect, test } from 'vitest';
+
+describe('BaseBranch.shortName', () => {
+	const testCases = [
+		// Cases with remoteName
+		{
+			branchName: 'refs/remotes/origin/feature/foo',
+			remoteName: 'origin',
+			expected: 'feature/foo',
+			description: 'full ref with remote'
+		},
```

Comment:
you could have some horror of a scenario where you have a remote called `origin/feature` and the branch name is only `foo`